dqc_us_rules/dqc_us_0004.py

```diff
@@ -52,7 +61,7 @@ def _values_unequal(val1, val2, dec_scale, margin_scale=2):
     """
     round_val1 = roundValue(val1, decimals=dec_scale)
     round_val2 = roundValue(val2, decimals=dec_scale)
-    margin_of_error = Decimal(margin_scale) * (_TEN ** Decimal(-dec_scale))
+    margin_of_error = Decimal(margin_scale) * (Decimal(10) ** Decimal(-dec_scale))
```

Comment:
Fair 'nuff.  Yeah, my original suggestion, then.  Remove the unused constant definition at the top of the file.